R/updateRecodeDB.R

```diff
@@ -88,11 +89,9 @@ updateRecodeDB <- function(newRecodes, oldValues, newValues = "newValues", recod
   ## if necessary, override & order
   updated_recode_list <- rbind(old_recode_list[!old_recode_list$oldValues %in% oldValues_conflicts$oldValues, ], newRecodes_manual)
   updated_recode_list <- updated_recode_list[order(updated_recode_list$oldValues), ]
-  recode_db[[name]] <- updated_recode_list
-
-  # overwrite existing excel sheet
-  writexl::write_xlsx(recode_db, path = newRecodeDBPath, col_names = TRUE)
+  recode_db[[ListName]] <- updated_recode_list
 
+  createRecodeDB(recodeListList = recode_db, directory = newDirectory, DBname = newDBname, fileType = fileType)
```

Comment:
Should overwrite the old DB, even if we don't overwrite a RecodeDB by default with `createRecodeDB()`. So some arguemnt would be necessary, that is set to override = TRUE in this case.